frontend/packages/git-service/src/services/__tests__/bitbucket-service.spec.ts

```diff
@@ -155,7 +155,7 @@ describe('Bitbucket Service', () => {
 
   it('should return exposed container port from dockerfile', () => {
     const gitSource: GitSource = {
-      url: 'https://bitbucket.org/jerolimov/nodeinfo',
+      url: 'https://bitbucket.org/christoph-jerolimov/nodeinfo',
```

Comment:
Old one works fine https://bitbucket.org/jerolimov/nodeinfo this one giving 404

Reply:
Reverted this change